include/arch/posix/linker.ld

```diff
@@ -25,19 +25,28 @@ SECTIONS
 /* Located in generated directory. This file is populated by the
  * zephyr_linker_sources() Cmake function.
  */
```

Comment:
Nit: This comment refers to the `#include` statement, so it might make more sense to have it directly before that.

Reply:
Moved.